src/layout/index.tsx

```diff
@@ -20,6 +20,7 @@ const StyledContent = styled(Content)`
 `
 const StyledBody = styled.div`
   padding: 24px;
+  padding-top: 0;
```

Comment:
you can write it shorter: `padding: 0 24px `

Reply:
you're right